src/frontmatter.xml

```diff
@@ -35,84 +35,87 @@
+            </ul>
+        </paragraphs>
+
+        <paragraphs>
+            <title>Tablets and Smartphones</title>
+            <p>MathBook XML documents like this text are <q>mobile-friendly</q>. The display adapts to whatever screen size or window size you are using. A math teacher will always recommend that you do not study from the small screen on a phone, but if it's necessary, this text gives you that option.</p>
+        </paragraphs>
+    </preface>
+
+    <preface xml:id="to-the-student">
+        <title>To the Student</title>
+
+        <p>You may think of this text as a <q>workbook</q>. Each chapter begins with activity sets, meant to serve as an introduction to a topic, and your instructor may assign activities to be done out of class as homework. In this way, you will come to class, having already studied the basics of a new topic, and ready to build on that new knowledge.</p>
+        <p>The activity sets are followed by a <q>gist</q> section. The gist assumes that you have completed the activities, and it provides a retelling of the basic concepts and techniques. This will be a good place for you to refer when studying the material for an exam or quiz.</p>
+        <p>Instead of just <em>reading</em> example problems, you will get to explore new ideas by working along <em>with</em> the examples. Most problems in the book allow you to manipulate dynamic graphs and receive immediate feedback by submitting your answers through WeBWorK online.</p>
+        <p>We wrote this text with students in mind. We know how busy the life of a student can be <mdash /> some working full-time jobs, some with children to care for, often while taking a full load of classes. We want this to be a safe learning space and reference that you can take anywhere you have an internet connection. Also, we are releasing this for free, to help lessen the cost of higher education.</p>
+        <p>We hope you find learning with this book to be an enjoyable and engaging experience!</p>
+
+    </preface>
+
+    <preface xml:id="to-the-instructor">
+
+        <title>To the Instructor</title>
+        <paragraphs>
+        <p>This book is a collection of activities, a digital workbook, that replaces traditional textbook reading with an interactive, guided-lesson approach to learning. The workbook covers common topics of precalculus that are otherwise delivered as passive reading assignments or lectures.</p>
+        <p>We hope this book will feel very natural to use. Each chapter begins with <em>activities</em> which introduce students to the basics of a topic. Activities can be assigned, graded and recorded through the use of <a href="http://webwork.maa.org/">WeBWorK</a>, an open-source online homework system. Many exercises include dynamic <a href="http://geogebra.org">GeoGebra</a> graphs for students to manipulate.</p>
```

Comment:
Instead of an "a" tag, the MBX is a "url" tag. "href" is still the right attribute.